authorizers/iconik/helper.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpBadRequestError, HttpInternalServerError, HttpUnauthorizedError } from '@errors/http';
+import { IconikService } from '@workflowwin/iconik-api';
+import { IconikParams } from '@workflowwin/iconik-api/src/index';
+import { IconikContext } from './interfaces/context';
+
+export async function getCallerAndOwner(iconikParams: IconikParams, callerId: string): Promise<IconikContext> {

```

Comment:
You name fn `getCallerAndOwner`, but function retures `IconikContext`. Would be better to name fn like `getIconikContext` to follow what it does.